pkgs/by-name/lo/louvre/package.nix

```
@@ -0,0 +1,72 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  meson,
+  ninja,
+  pkg-config,
+  fontconfig,
+  icu,
+  libdrm,
+  libGL,
+  libinput,
+  libX11,
+  libXcursor,
+  libxkbcommon,
+  mesa,
+  pixman,
+  seatd,
+  srm-cuarzo,
+  udev,
+  wayland,
+  xorgproto,
+}:
+stdenv.mkDerivation (self: {
+  pname = "louvre";
+  version = "1.1.0-1";
+  rev = "v${self.version}";
+  hash = "sha256-HwvX0ykl2+4MBcIixmEknFtsB0QC4w1QDzQz1589bl0=";
+
+  src = fetchFromGitHub {
+    inherit (self) rev hash;
+    owner = "CuarzoSoftware";
+    repo = "Louvre";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+  ];
+
+  buildInputs = [
+    fontconfig
+    icu
+    libdrm
+    libGL
+    libinput
+    libX11
+    libXcursor
+    libxkbcommon
+    mesa
+    pixman
+    seatd
+    srm-cuarzo
+    udev
+    wayland
+    xorgproto
+  ];
+
+  outputs = ["dev" "out"];
```

```suggestion
  outputs = [ "dev" "out" ];
```
I don't know why but we like spaces in nixpkgs :shrug: